src/drivers/Bma421.h

```diff
@@ -35,15 +36,28 @@ namespace Pinetime {
       bool IsOk() const;
       DeviceTypes DeviceType() const;
 
+      bool getAndClearWristTiltInterrupt() {
```

Comment:
[Functions should use PascalCase](https://github.com/InfiniTimeOrg/InfiniTime/blob/main/doc/coding-convention.md).